apps/sports-helsinki/src/domain/event/eventHero/EventHero.tsx

```diff
@@ -41,9 +44,11 @@ export interface Props {
   superEvent?: SuperEventResponse;
 }
 
+// eslint-disable-next-line sonarjs/cognitive-complexity
```

Comment:
Should not be needed.

Reply:
Removed in 47ae930be77334af2ac7194f56538794eb4bb3ca